src/rgw/driver/sfs/sqlite/dbconn.h

```diff
@@ -298,7 +298,7 @@ class DBConn {
   Storage storage;
 
  public:
-  sqlite3* sqlite_db;
+  sqlite3* first_sqlite_conn;
```

Comment:
Better to explicitly initialise this pointer to `nullptr`, just in case.
Because of the new code added:
```c++
     if (first_sqlite_conn == nullptr) {
        first_sqlite_conn = db;
      }
```